pkg/kubelet/kubelet_network.go

```diff
@@ -55,26 +55,28 @@ func (kl *Kubelet) providerRequiresNetworkingConfiguration() bool {
 }
 
 // updatePodCIDR updates the pod CIDR in the runtime state if it is different
-// from the current CIDR.
-func (kl *Kubelet) updatePodCIDR(cidr string) error {
+// from the current CIDR. Return true if pod CIDR is actually changed.
+func (kl *Kubelet) updatePodCIDR(cidr string) (bool, error) {
```

Comment:
Also note the change here. We need this return value to know if pod CIDR is actually changed or not.